pennylane/expval/__init__.py

```diff
@@ -50,5 +50,44 @@
     expval/cv
 """
 
-from .qubit import *
-from .cv import *
+from pennylane.qnode import QNode, QuantumFunctionError
+
+from .qubit import * #pylint: disable=unused-wildcard-import,wildcard-import
+from .cv import * #pylint: disable=unused-wildcard-import,wildcard-import
```

Comment:
Instead of having a wildcard import (which I've never really liked due to the ambiguity), perhaps we should explicitly import each expval here

Reply:
I thought it would be more elegant to control what is imported here through `__all__` in the sub-modules, which I anyway have to set. Repeating the same list here looked like code duplication to me.

Reply:
:+1: